SPECS/core-packages/core-packages.spec

```diff
@@ -83,6 +80,10 @@ Requires:       vim
 Requires:       wget
 Requires:       xz
 Requires:       zlib
+%ifarch x86_64
+# Temporarily exclude aarch64 from including the microsoft repo until content is available in the repo
+Requires:       mariner-repos-microsoft
+%endif
```

Comment:
You should move this section back up to where it was right below `Requires: mariner-repos`, it makes more sense there alphabetically, the linter just gets confused by the %ifarch.

Reply:
Done